src/MultiFormGoals/MultiFormGoal/Shortcode.php

```diff
@@ -27,8 +27,6 @@ public function addShortcode() {
 	 * @since 2.9.0
 	 **/
 	public function renderCallback( $attributes ) {
-		error_log( serialize( $attributes ) );
```

Comment:
Why is this being removed?

Reply:
It was a part of the previous fix which was merged into this PR.